packages/node/src/node.ts

```diff
@@ -8,6 +8,9 @@ import { Base } from "@logtail/core";
 
 import { getStackContext } from "./context";
 
+const CONTEXT_OBJECT_MAX_DEPTH_DEFAULT : number = 20;
```

Comment:
I'm not sure if 20 is high enough. Stack size limits are usually in thousands, so I would bump this to 100 or so, even though logs that size are not going to be very practical.

Reply:
yep I'll set it to 100.

from backend service point of view, I'agree this is not very practical/realist. But if there is no technical issue let's do it.

Reply:
I'll set it to 50 because with 100 I've another issue to fix:
```
  console.error packages/core/dist/cjs/base.js:137
    Error: Too deep objects in depth 101
        at Encoder.doEncode (C:\WORK\perso\logtail-js\packages\node\node_modules\@msgpack\msgpack\src\Encoder.ts:42:13)
        at Encoder.encodeMap (C:\WORK\perso\logtail-js\packages\node\node_modules\@msgpack\msgpack\src\Encoder.ts:286:14)
        at Encoder.encodeObject (C:\WORK\perso\logtail-js\packages\node\node_modules\@msgpack\msgpack\src\Encoder.ts:197:12)
```